fpmsyncd/routesync.cpp

```diff
@@ -911,6 +911,11 @@ string RouteSync::getNextHopGw(struct rtnl_route *route_obj)
         if (addr)
         {
             char gw_ip[MAX_ADDR_SIZE + 1] = {0};
+            if (nl_addr_get_prefixlen(addr) > IPV4_MAX_BITLEN
+                && nl_addr_get_family(addr) == AF_INET)
```

Comment:
what's this case? zebra issue?

Reply:
Yes, Zebra sends ipv6 link-local as the nexthop in case of RFC5549.

Reply:
Is this not confusing i.e sending prefix len more than ipv4 range with address family AF_INET?

Reply:
do we have a fix plan for this in zebra?

Reply:
Reverting this change as of now, Planning for a new PR later.